resources/eventing/charts/controller/values.yaml

```diff
@@ -0,0 +1,32 @@
+# override name to avoid collision with knative eventing resources
+nameOverride:
+replicaCount: 1
+serviceAccount:
+  # name defines optionally another name than the default name for the service account
+  name: ""
+  annotations: ""
+
+
+clusterRole:
+  # name defines optionally another name than the default name for the cluster role
+  name: ""
+
+image:
+  # name is the name of the container image for the eventing-controller
+  name: "eventing-controller"
+  # tag is the container tag of the eventing-controller image
+  tag: "PR-9868"
+  # pullPolicy is the pullPolicy for the eventing-controller image
+  pullPolicy: "IfNotPresent"
+
+podSecurityContext:
+
+bebSecret:
+  nameSuffix: "-beb-oauth2"
+
+metrics:
+  service:
+    nameSuffix: "-metrics"
+
+#TODO(sayanh): Remove this when knative-eventing chart is removed
```

Comment:
Can we get rid of this and just log a GitHub issue or create a follow-up on your board to do this?

Reply:
Changed